uint8 BatteryLevel
uint8 BatteryCharging
uint8 ConnStrengthVSC
uint8 ConnStrengthSRC
